project/frontend/src/components/CuisineAutocomplete.js

```diff
@@ -19,7 +19,9 @@ import TextField from '@material-ui/core/TextField';
  */
 function inputIsNewAndValid(input, options) {
   const trimmedInput = input.trim();
+  const maxNumCuisines = 10;
   return (
+    options.length < maxNumCuisines &&
```

Comment:
does this mean if we get fewer than 10 cuisines that are available that we can list, this would be invalid input?

Reply:
oh oops yes you're right I need to fix this to use the selected options

Reply:
👍 yeah it should be display max(numCuisines, 10) options